src/code/z_player_lib.c

```diff
@@ -1637,7 +1637,7 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
     }
 }
 
-u32 func_80091738(PlayState* play, u8* segment, SkelAnime* skelAnime) {
+u32 Player_InitDrawPause(PlayState* play, u8* segment, SkelAnime* skelAnime) {
     s16 linkObjectId = gLinkObjectIds[(void)0, gSaveContext.linkAge];
```

Comment:
I totally get what this name is going for, but wondering if we can improve it. I get its related to drawing in that its loading assets and initializing skelanime, but personally wouldnt call this as "init for draw". It feels more logic-side to me.

Especially considering the larger point of this function is the dma-ing thats happening (and the size that gets returned), I wonder if the name should be more centered around that. or maybe just `Player_InitForPause`, removing draw and making it more general "init things for the pause menu"

Reply:
To explain why I initially picked "InitDrawPause": If you look at `Player_DrawPause`, that's where the "use" side of the buffer this sets up is, so I just added "Init" to that name <img src="https://cdn.discordapp.com/emojis/628286334066491412.webp?size=56&quality=lossless" alt="4Head" height="24">

fwiw `Player_InitForPause` sounds to me like it would change something in the player system, but afaik pause menu does nothing to the player (`Player`)

Reply:
yea the intent is "init player related things for drawing". But thats fair either way.
I cant put into words why "init draw" makes my brain hurt. I know it can be taken to understand "init the things that will be used for drawing", but it just doesnt sound right lol

I think a name more centered on the data dma it does is still potentially viable, `Player_InitPauseData` or something, with skelanime being lumped in there as a side thing, idk

It also loads gameplay keep which isnt even player centric. Which makes me kinda doubt the player prefix at all, but i guess a majority of the function is player related

Reply:
Afaik gkeep is only (partially) loaded for the purpose of drawing player, for example for `gHylianShieldDesignTex` and other textures (and `gLinkPauseChildJointTable` and family apparently) (actually maybe all the ~0x2000 first bytes of gkeep, though 0x5000 are kept and idk what/if the next 0x3000 are used for)

The whole buffer is named `playerSegment` (`PauseContext.playerSegment`) so if we move away from player at all (but at this time I don't see a reason we should), it's something to rename too

`Player_PauseInitPlayerSegment` ? bleh

Currently:
```
Player_InitDrawPause
Player_DrawPause
Player_DrawPauseImpl
```

Could also do:
```
Player_PauseInitSegment
Player_PauseDraw
Player_PauseDrawImpl
```
or something

Reply:
The second list doesn't seem too bad. Could also `Player_InitPauseData` and `Player_InitDrawPause` and do something like `Player_InitDrawPauseData/Player_InitPauseDrawData`?

Reply:
i am not opposed to `Player_InitPauseDrawData` tbh, it gets rid of my issue of "init" and "draw" being next to each other. it tells you that its initializing draw data

Reply:
`Player_InitPauseDrawingData` maybe to elimate the confusion of "init pause, draw the data", but that also doesnt really make sense anyway and think itd be clear already

Reply:
I'm fine with `Player_InitPauseDrawData`

do I keep `Player_DrawPause` and `Player_DrawPauseImpl`,
or change to `Player_PauseDraw` and `Player_PauseDrawImpl`

Reply:
Id say DrawPause still, to stay in line with things like `Player_DrawGameplay`. think its fine if the data init func doesnt have the same scheme